src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.Machines.Bulldozer;
+import core.basesyntax.Machines.Excavator;
+import core.basesyntax.Machines.Machine;
+import core.basesyntax.Machines.Truck;
+
 public class MainApp {
+  public static void main(String[] args) {
+
+    Truck truck = new Truck();
+    Excavator excavator = new Excavator();
+    Bulldozer bulldozer = new Bulldozer();
```

Comment:
Use abstract references instead of specific ones where possible to take advantage of polymorphism. Instead of having a 'Truck' reference for 'truck', use 'Machine' as the reference type for all these objects.